tests/main/completion/task.yaml

```diff
@@ -15,6 +23,7 @@ prepare: |
     expect -d -f key.exp0
 
 restore: |
+    chattr -i /var/cache/snapd/names
```

Comment:
Shouldn't it be removed since prepare created it?

Reply:
starting snapd created it. Maybe the test should stop snapd, move the old names aside, then do this, and restore it on restore?

Reply:
Or just remove it again, which sounds easier.. the test just removed it anyway, and this is the one test that cares the most about this file being right.